.github/workflows/validate.yml

```diff
@@ -262,6 +262,12 @@ jobs:
           ghc-version: ${{ matrix.ghc }}
           cabal-version: latest # default, we are not using it in this job
 
+      - name: Install cabal-plan
+        run: |
+          cd $(mktemp -d)
```

Comment:
The `cd` step is surprising, which purpose does it serve?

Reply:
I guess it's tradition since Oleg started doing it in 877ffa475d4ddd8d75c0c9467d21791dae982b00 :D Now, seriously, I want to make sure cabal is not influenced by the project in the current directory.

Reply:
Understood!